plugins/TelescopeControl/src/TelescopeControl.cpp

```diff
@@ -180,9 +185,25 @@ void TelescopeControl::init()
 			text = q_("Move telescope #%1 to the point currently in the center of the screen").arg(i);
 			StelAction* actionSlewDir = addAction(name, section, text, slewDir, "map()", shortcut);
 			slewDir->setMapping(actionSlewDir, i);
+
+			// "Sync to object" commands
+			name = syncActionId.arg(i);
+			shortcut = QString("Ctrl+Shift+%1").arg(i);
+			text = q_("Sync telescope #%1 position to selected object").arg(i);
+			StelAction* actionSyncObj = addAction(name, section, text, syncObj, "map()", shortcut);
+			syncObj->setMapping(actionSyncObj, i);
+
+			// "Abort Slew" commands
+			name = abortSlewActionId.arg(i);
+			shortcut = QString("Ctrl+Shift+Alt+%1").arg(i);
+			text = q_("Abort last slew command of telescope #%1").arg(i);
+			StelAction* actionAbortSlew = addAction(name, section, text, abortSlew, "map()", shortcut);
+			abortSlew->setMapping(actionAbortSlew, i);
```

Comment:
@gzotti Should we document the new keyboard shortcuts somewhere?

Reply:
Yes, in the Stellarium User Guide and Help window of plugin

Reply:
I hope you are already writing an ASCOM section in the Stellarium User Guide (/guide/plg_atTheTelescope.tex). If the new hotkeys only apply to ASCOM telescopes, it should go there, else into the general plugin description. The hotkey actions themselves should be automatically listed in the F1 help screen. 

Speaking of User Guide: What do we do with StellariumScope? Is it obsolete now, or are there still scenarios for its use? (Networked solutions?) I am just not enough of an observer currently to see all the various applications and variants of using Stellarium with a gazillion of telescope mounts. 

While I see the "map()" function called: I have recently seen that the QSignalMapper is deprecated/obsolete, and we should rather get rid of it. In some Qt forum I found descriptions how to update C++ code with lambda functions etc, which is fragile territory for me. Not sure if we can ask you to do that for the telescope plugin, or @alex-w  do you know already what to do? They are used in TelescopeControl and Ocular plugins, those are the ones I currently use the least, personally, so I may miss something when I just change and make a few tests.

Reply:
@gzotti I’ve tried use lambda for change the QSignalMapper but without success, because lambda’s cannot use QString. I didn’t make a “wrapper” for it.

Reply:
@alex-w  Ouch. Is it possible to use QString.toByteArray().data() or something like that?

Reply:
@gzotti I think StellariumScope is obsolete now. Since I have implemented the new telescope control sync and abort slew commands, all the additional functionality which was provided by StellariumScope is covered.

Reply:
> @alex-w Ouch. Is it possible to use QString.toByteArray().data() or something like that?

I don’t know because I didn’t touch lambdas after previous experiment